### DISCUSSION TITLE
Hide Stash IDs

### DISCUSSION THREAD

**Discussion by nathanstreetx** [stashapp/stash#5463] [Q&A] [answered]
Is there any way to hide the Stash IDs from view across all pages (Performers, Studio's etc.)

To be clear, I don't want to remove the Stash IDs, as they are needed for scraping. I just want to hide them from view.

![image](https://github.com/user-attachments/assets/600e8d1d-1769-419e-9a16-201f70ac0010)

**Comment by neru2132** [comment, marked as answer]
You can enable custom CSS in the Interface settings. I'm not an expert myself, but I believe the following should get the job done:
`.stash_ids {
  display: none !important;
}`

**Comment by nathanstreetx** [comment]
@neru2132 Perfect, that worked. Thank you :)

**Comment by nathanstreetx** [comment]
@neru2132 Perfect, that worked. Thank you :)
